docs/scripting/trust_scripts/sys.upgrade_log.mdx

```diff
@@ -57,7 +57,6 @@ Get a range with count:<count> and start:<offset>
 #### Script
 
 ```
->>specs
```

Comment:
This diff does not line up with the PR title

Reply:
The context is kb was using `kbeeb.specs` to get the subscript output, and accidentally left the CLI call to `kbeeb.specs` in the article. I'm happy for this to get removed in-line with the PR title

Reply:
oh haha. I get it.